compiler/rustc_codegen_ssa/src/back/link.rs

```diff
@@ -403,6 +418,14 @@ fn link_rlib<'a>(
         ab.add_file(&trailing_metadata);
     }
 
+    // Add all bundled static native library dependencies.
+    // Archives added to the end of .rlib archive, see comment above for the reason.
+    if flavor == RlibFlavor::Normal && sess.opts.unstable_opts.packed_bundled_libs {
```

Comment:
Redundant condition, `packed_bundled_libs` should alway be empty if the combination of options is not right.